src/main/java/org/frc5687/powerup/robot/subsystems/Arm.java

```diff
@@ -87,7 +87,7 @@ protected double returnPIDInput() {
 
     @Override
     protected void usePIDOutput(double output) {
-        DriverStation.reportError("Arm PID output at " + output, false);
+        SmartDashboard.putNumber("Arm PID output at ", output);
```

Comment:
I'm not the biggest fan of unorganized keys. What about `Arm/PID/output` for the key.